csaf_files/IT/white/2024/va-24-201-01.json

```diff
@@ -158,7 +158,7 @@
                   "product_id": "CSAFPID-0078"
                 },
                 "category": "product_version_range",
-                "name": "all versions"
+                "name": "<4.8.4"
```

Comment:
See above comment re: version:
https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31231-branches-type---name-under-product-version

Reply:
I agree with the above change.